pkg/diagnostics/resiliency_monitoring_test.go

```diff
@@ -407,7 +407,7 @@ func TestResiliencyActivationsCountMonitoring(t *testing.T) {
 					})
 				}
 				// let the circuit breaker to go to half open state (5x cb timeout) and then return success to close it
-				time.Sleep(500 * time.Millisecond)
+				time.Sleep(1000 * time.Millisecond)
```

Comment:
I think its just a timing issue for this: https://github.com/dapr/dapr/actions/runs/9570009389/job/26384075690?pr=7818